projects/observability/src/shared/dashboard/widgets/charts/cartesian-widget/cartesian-widget-renderer.component.ts

```diff
@@ -58,13 +58,16 @@ export class CartesianWidgetRendererComponent<TSeriesInterval> extends Interacti
     return this.model.getDataFetcher().pipe(
       tap(fetcher => {
         this.fetcher = fetcher;
+        const defaultInterval =
+          this.model.defaultInterval?.value !== undefined ? this.model.defaultInterval.getDuration() : undefined;
```

Comment:
`this.model.defaultInterval?.value !== undefined ? this.model.defaultInterval.getDuration() : undefined;`

One weird thing i noticed is that our model library creates empty model object even when we don't define them in the model json. It means I can't used something like this ` this.model.defaultInterval?.getDuration()`

Reply:
Not by default it shouldn't - the newly added property requests a default instance via the `defaultModelClass` key:

```
defaultModelClass: TimeDurationModel
```
from docs:
```ts
export interface ModelModelPropertyTypeInstance extends ModelPropertyTypeInstance {
    /**
     * If the property is undefined, by default, use this class
     */
    defaultModelClass?: Constructable<object>;
}
```

Reply:
So i can safely remove this property then?

Reply:
yep, it's there explicitly to create a default instance if none is provided, which is the opposite of what you want.

Reply:
done